modules/system/ServiceProvider.php

```diff
@@ -170,6 +171,24 @@ protected function registerPrivilegedActions()
         }
     }
 
+    /*
+     * Register class aliases for the plugins.
+     */
+    protected function registerClassAliases()
+    {
+        $plugins = PluginManager::instance()->getRegistrationMethodValues('registerClassAliases');
+        foreach ($plugins as $plugin => $aliases) {
+            if (!is_array($aliases)  || empty($aliases)) {
+                continue;
+            }
+            foreach ($aliases as $real => $alias) {
+                if (!class_exists($alias)) {
```

Comment:
Why have this check for class_exists?

Reply:
@LukeTowers plugin:refresh in the tests was complaining the class already existed.

Reply:
@LukeTowers I don't believe you can alias a class that exists, from memory.

Reply:
Gotcha, I thought it was just going to ignore it and move on with life but if it complains that makes sense.